src/anomalib/models/efficient_ad/lightning_model.py

```diff
@@ -59,10 +59,10 @@ class EfficientAd(AnomalyModule):
         model_size (str): size of student and teacher model
         lr (float): learning rate
         weight_decay (float): optimizer weight decay
-        padding (bool): use padding in convoluional layers
-        pad_maps (bool): relevant if padding is set to False. In this case, pad_maps = True pads the
-            output anomaly maps so that their size matches the size in the padding = True case.
+        padding (bool): use padding in convoluional layers of the student/teacher architecture
```

Comment:
spelling/grammar nitpick
```suggestion
        padding (bool): use padding in the convolutional layers of the student/teacher architecture
```